src/WebManager/APIResponse.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace WebManager
+{
+    public class APIResponse
+    {
+        public string content { set; get; } 
```

Comment:
We typically use PascalCase for class-level declarations in C# e.g.: `content` -> `Content`. Use this for all properties in this class.